.gitignore

```diff
@@ -9,3 +9,7 @@ pubspec.lock
 *~
 .pub/
 bazel-*
+.idea/
+.gitignore
+.dart_tool/
+*.g.dart
```

Comment:
Why this?

Reply:
I'm still trying to decide if I want to commit the generated files directly or not. build.yaml isn't working very well since this library uses built_value (generating .g.dart files) and pageloader will also generate .g.dart files. 

Going to have to loop in the build_runner folks to get some advice.